f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_acquisition_package.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?><database>
-    <element label="DAPPS:Acquisition Package" max_length="40" name="x_g_dis_atat_acquisition_package" sizeclass="163" type="collection">
+    <element db_object_id="3a80dd5e971121107b1372571153af2b" label="DAPPS:Acquisition Package" max_length="40" name="x_g_dis_atat_acquisition_package" type="collection">
         <element label="ACOR" max_length="32" name="acor" reference="x_g_dis_atat_contacts" type="reference"/>
+        <element label="Address" max_length="32" name="address" reference="x_g_dis_atat_dapps_address_table" type="reference"/>
```

Comment:
It's not clear to me what this new column `Address` in `Acquisition Package` table is for.  New table `Address Table` has a reference to `Acquisition Package`, so _many_ addresses can be associated with a single package.  But this column would be used to associate a single address to a package; this appears to be redundant.